### ISSUE TITLE
The categories of patterns from the previous version have disappeared

### ISSUE THREAD

**Issue by miaskor** [iluwatar/java-design-patterns-vuepress-web#38]
![image](https://user-images.githubusercontent.com/104252450/202926517-834c2d24-491c-4aa5-b163-6f5ac2cd5606.png)
Previously, it was possible to select patterns by category: structural, behavioral, and so on

**Comment by iluwatar** [comment]
You are right, this is not possible (yet) on the current website. We will add it.
